palace/drivers/drivensolver.cpp

```diff
@@ -365,11 +355,17 @@ ErrorIndicator DrivenSolver::SweepAdaptive(SpaceOperator &spaceop, PostOperator
     step++;
     omega += delta_omega;
   }
+  BlockTimer bt0(Timer::POSTPRO);
+  SaveMetadata(prom.GetLinearSolver());
   return indicator;
 }
 
 int DrivenSolver::GetNumSteps(double start, double end, double delta) const
 {
+  if (end < start)
```

Comment:
Maybe `<=`? Skips the logic if driving a single frequency.

Reply:
My thought was that the logic below works fine for `start = end`, so no need to bypass it.